shared/src/types/admin/Survey.ts

```diff
@@ -0,0 +1,24 @@
+import { RecipientMainLanguage } from './Recipient';
+
+export const SURVEY_FIRETORE_PATH = 'surveys';
+
+export enum SurveyStatus {
+	New = 'new',
+	Sent = 'sent',
+	Started = 'started',
+	Completed = 'completed',
+	Missed = 'missed',
+}
+
+export type Survey = {
+	recipient_name: string; // used to address the recipient in the form. E.g. Hello XYZ.
```

Comment:
Can't we just read this from the recipient doc?

Reply:
Yes, valid point.
The thought behind it was that the user for this can only access the survey data (since we are sending around the credentials for those in the urls) and to limit therefore all the security rules to only this document to keep things relatively closed. Wdyt?